task/buildah/0.3/MIGRATION.md

```diff
@@ -0,0 +1,7 @@
+# Migration from 0.2 to 0.3
+
+Version 0.3:
+
+Removes references to `jvm-build-service`
+* Removes `analyse-dependencies-java-sbom` step
+* Removes `SBOM_JAVA_COMPONENTS_COUNT` and `JAVA_COMMUNITY_DEPENDENCIES` results
```

Comment:
It looks like these results were not referenced anywhere in our pipelines, so the migration generally shouldn't require any action from users?

Let's call that out explicitly, similarly to e.g. https://github.com/konflux-ci/build-definitions/blob/main/task/sast-snyk-check/0.3/MIGRATION.md